test/json_test.c

```diff
@@ -43,7 +43,7 @@ static void helper_cleanup(struct helper *h)
     BIO_free_all(h->mem_bio);
     h->mem_bio = NULL;
 
-    if (!h->init) {
+    if (h->init) {
```

Comment:
nit: shouldn't this be if (h->init != 0)

Reply:
My understanding of our conventions is that it's OK to omit this for things which are "bools", even though we don't have an explicit int type. Which is why you can write `if (!SSL_foo())` rather than `if (SSL_foo() == 0)`.

OTOH, for some variable which can admit values other than `{0, 1}`, yes, this change should be made.